spec/mem/tree_borrows/tree.md

```diff
@@ -46,12 +62,13 @@ impl Node {
         node_relation: NodeRelation,
         access_kind: AccessKind,
         offset_in_alloc: Size,
-        size: Size
+        size: Size,
     ) -> Result {
+        let protected = self.protected != Protected::No;
```

Comment:
This is wrong. This used to be 
```
let protected = if let Protected::Yes(call_id) = self.protected { active_calls.contains_key(call_id)} else { false }; 
```
two days ago when I looked at it, which was (more) correct. Please change it back to what it was before everywhere. Protectors are only relevant when their calls are active.

It is still not fully correct, see the comment above.

Reply:
The current `release_protector` will make the protected node
`self.protected = Protected::No`

Does it make sense?

Reply:
Ah, yes, that works.